hnn/qt_spike.py

```diff
@@ -90,7 +90,7 @@ def getdspk(spikes, extinputs, tstop):
             haveinputs = True
     for ty in dhist.keys():
         dhist[ty] = np.histogram(dhist[ty], range=(0, tstop),
-                                 bins=int(tstop / binsz))
+                                 bins=ceil(tstop / binsz))
```

Comment:
why not just used the default bins of matplotlib?

Reply:
Good idea. Please contribute a PR for that that supercedes this.